src/main/java/com/mraof/minestuck/data/recipe/MinestuckCombinationsProvider.java

```diff
@@ -144,7 +151,47 @@ protected void buildCraftingRecipes(Consumer<FinishedRecipe> consumer)
+		CombinationRecipeBuilder.of(Items.MEDIUM_AMETHYST_BUD).input(Items.SPYGLASS).and().input(Items.AMETHYST_SHARD).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.SMALL_AMETHYST_BUD).input(Items.SPYGLASS).or().input(Items.MEDIUM_AMETHYST_BUD).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.CALCITE).input(Items.AMETHYST_SHARD).or().input(Items.STONE).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.CANDLE).input(Items.BOOK).and().input(Items.TORCH).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.COBBLED_DEEPSLATE).input(Items.COBBLESTONE).and().input(Items.CHARCOAL).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.DEEPSLATE).input(Items.STONE).or().input(Items.INK_SAC).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.POINTED_DRIPSTONE).input(Items.CHAINMAIL_BOOTS).and().input(Items.ARROW).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.POINTED_DRIPSTONE).input(Items.DIAMOND_BOOTS).or().input(Items.ARROW).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.POINTED_DRIPSTONE).input(Items.IRON_BOOTS).and().input(Items.ARROW).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.POINTED_DRIPSTONE).input(Items.NETHERITE_BOOTS).or().input(Items.ARROW).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.POINTED_DRIPSTONE).input(Items.LEATHER_BOOTS).and().input(Items.ARROW).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.POINTED_DRIPSTONE).input(Items.GOLDEN_BOOTS).or().input(Items.ARROW).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.POINTED_DRIPSTONE).input(Items.DRIPSTONE_BLOCK).or().input(Items.ARROW).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.LIGHTNING_ROD).input(Items.REDSTONE).and().input(Items.COPPER_INGOT).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.DRIPSTONE_BLOCK).input(Items.STONE).and().input(Items.BIG_DRIPLEAF).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.POWDER_SNOW_BUCKET).input(Items.ANVIL).or().input(Items.ICE).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Blocks.POWDER_SNOW).input(Items.WHITE_CONCRETE_POWDER).and().input(Items.ICE).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.ROOTED_DIRT).input(Items.DIRT).and().input(Items.JUNGLE_SAPLING).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.SMOOTH_BASALT).input(Items.INK_SAC).or().input(Items.STONE).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.TINTED_GLASS).input(Items.INK_SAC).or().input(Items.FERMENTED_SPIDER_EYE).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.TUFF).input(Items.FLINT_AND_STEEL).or().input(Items.BIRCH_WOOD).buildFor(consumer, Minestuck.MOD_ID);
```

Comment:
I assume this combination is supposed to represent the volcanic ash tuff is made from, if so nice thinking! Its nitpicky of me so if you dont want to change it thats fine but in my opinion this combination would result in some sort of ash block as opposed to rock made from ash

Reply:
i'm going for the explanation on how alchemy works that is more based around ideas then actual physical combination of objects as i think it makes alchemy unpredictable and a little interesting, although for reasons of being incredibly stand out to the more "Crafting system +" type alchemy we have it can stand to be changed